apps/cwl_to_slack/app.py

```diff
@@ -0,0 +1,59 @@
+import json
+import requests
+import boto3
+from botocore.exceptions import ClientError
+
+
+def handler(event, context):
+    """
+    main lambda function
+    """
+
+    secrets = json.loads(get_secret()['SecretString'])
+    slack_url = secrets['slack_webhook_url']
+    slack_channel = secrets['slack_alert_channel']
+
+    alert_message = json.loads(event['Records'][0]['Sns']['Message'])
+    alarm_name = alert_message['AlarmName']
+    new_state = alert_message['NewStateValue']
+    reason = alert_message['NewStateReason']
+
+    alert_url = 'https://console.aws.amazon.com/cloudwatch/home?region=us-east-1#alarm:alarmFilter=ANY;name={}'.format(
+        alarm_name)
+    slack_message = "<{alert_url}|{alarm_name}> state is now {new_state}: {reason}".format(
```

Comment:
use [f-strings](https://www.python.org/dev/peps/pep-0498/) since you are python 3.6+